content/terraform/v1.12.x/docs/language/block/ephemeral.mdx

```diff
@@ -13,7 +13,24 @@ The `ephemeral` block declares a temporary ephemeral resource that only exists d
 
 To learn more about the `ephemeral` block and its unique lifecycle and provisioning order, refer to [Sensitive data in State](/terraform/language/state/sensitive-data).
 
-## Refer to ephemeral resources
+### Lifecycle
```

Comment:
I realized this page should really live on the ephemeral reference. I know it's conceptual, but splitting it out into it's own page just feels silly.